<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWin</class>
 <widget class="QMainWindow" name="MainWin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1048</width>
    <height>770</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PendulumGUI</string>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QWidget" name="UpperPart" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="1">
        <widget class="QGroupBox" name="ICbox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>150</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="sizeIncrement">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="baseSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="title">
          <string>Initial Conditions:</string>
         </property>
         <widget class="QLineEdit" name="x0in">
          <property name="geometry">
           <rect>
            <x>120</x>
            <y>29</y>
            <width>68</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>90.0</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="x1in">
          <property name="geometry">
           <rect>
            <x>120</x>
            <y>63</y>
            <width>67</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>0.0</string>
          </property>
         </widget>
         <widget class="QLabel" name="x0label">
          <property name="geometry">
           <rect>
            <x>32</x>
            <y>31</y>
            <width>92</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>Initial Angle(deg):</string>
          </property>
         </widget>
         <widget class="QLabel" name="x1label">
          <property name="geometry">
           <rect>
            <x>4</x>
            <y>61</y>
            <width>121</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>Initial Velocity(deg/sec):</string>
          </property>
         </widget>
         <widget class="QPushButton" name="EvalTraj">
          <property name="geometry">
           <rect>
            <x>11</x>
            <y>99</y>
            <width>177</width>
            <height>23</height>
           </rect>
          </property>
          <property name="text">
           <string>Evaluate</string>
          </property>
         </widget>
        </widget>
       </item>
       <item row="0" column="5">
        <widget class="QLabel" name="label">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap>Pund1_p.svg</pixmap>
         </property>
        </widget>
       </item>
       <item row="0" column="3" rowspan="2">
        <widget class="QWidget" name="MPcanvas4PP" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>350</width>
           <height>300</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>400</width>
           <height>300</height>
          </size>
         </property>
         <property name="baseSize">
          <size>
           <width>396</width>
           <height>396</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="0" column="6">
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="0">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QScrollArea" name="MPcanvasScroll">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOn</enum>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QAbstractScrollArea::AdjustToContents</enum>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>1011</width>
         <height>618</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QWidget" name="MPcanvas4traj" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Ignored">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>600</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1048</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
